demos/src/demo.scss

```diff
@@ -45,3 +45,5 @@ $icon-list: ('grid', 'list');
 	$icons: $icon-list,
 	$theme: $custom-theme
 );
+
+@include oForms();
```

Comment:
I think this outputs form css twice because of the silent mode variable above, would be good to remove that

Reply:
Done 👍